kogito-usertasks-custom-lifecycle-springboot/README.md

```diff
@@ -12,9 +12,33 @@ This example shows
 
 * working with user tasks
 * four eye principle with user tasks
-	
-	
+
+* Diagram	
 <p align="center"><img width=75% height=50% src="docs/images/process.png"></p>
+	
+Diagram Properties
+<p align="center"><img src="docs/images/diagramProperties.png"></p>
+
+* Diagram Properties
+<p align="center"><img src="docs/images/diagramProperties2.png"></p>
+
+* Diagram Properties
+<p align="center"><img src="docs/images/diagramProperties3.png"></p>
+
+* First Line Approval
+<p align="center"><img src="docs/images/firstLineApprovalUserTask.png"></p>
+
+* First Line Approval
+<p align="center"><img src="docs/images/firstLineApprovalUserTask2.png"></p>
+
+* First Line Approval
+<p align="center"><img src="docs/images/firstLineApprovalUserTaskAssignments.png"></p>
+
+* Second Line Approval
+<p align="center"><img src="docs/images/secondLineApprovalUserTask.png"></p>
+
+* Second Line Approval
+<p align="center"><img src="docs/images/secondLineApprovalUserTaskAssignments.png"></p>
 
 In addition this quickstart also shows how to use custom life cycle for user tasks that uses custom life cycle phases
```

Comment:
I would put a quick note about that at the beginning of the file so that the user knows there is a difference.
Then we can let the explanations here

Reply:
Hi @radtriste can you explain?

Reply:
Put first line at the top of the README (as it is commented)

And change line 45:
`To learn more about custom lifecycle, look at the following classes:`

Reply:
fixed @radtriste